indra/llwindow/llkeyboardheadless.h

```diff
@@ -33,20 +33,15 @@ class LLKeyboardHeadless : public LLKeyboard
 {
 public:
     LLKeyboardHeadless();
-    /*virtual*/ ~LLKeyboardHeadless() {};
+    ~LLKeyboardHeadless() {};
 
-#ifndef LL_SDL
-    /*virtual*/ bool    handleKeyUp(const U16 key, MASK mask) { return false; }
-    /*virtual*/ bool    handleKeyDown(const U16 key, MASK mask) { return false; }
-#else
-    /*virtual*/ bool    handleKeyUp(const U32 key, MASK mask) { return false; }
-    /*virtual*/ bool    handleKeyDown(const U32 key, MASK mask) { return false; }
-#endif
-    /*virtual*/ void    resetMaskKeys();
-    /*virtual*/ MASK    currentMask(bool for_mouse_event);
-    /*virtual*/ void    scanKeyboard();
```

Comment:
The `/*virtual*/` comments here are artifacts from the olden days before the `override` keyword was added to C++.  They were used to signal the reader that these were virtual methods in the base class that the derived class was trying to override.  These days, please use the `override` keyword because it allows the compiler to enforce the override signature is correct and corresponds to a base method that really is virtual.